test/lambda.py

```diff
@@ -0,0 +1,22 @@
+import boto3
+import botocore
+import json
+import os
+
+s3 = boto3.resource('s3', region_name = 'us-east-1')
+
+bucket_name = 'inbox-bucket-test'
+bucket_name1 = 'inbox-bucket-testresized'
+
+bucket = s3.Bucket(bucket_name)
+print("created source bucket\n")
+
+bucket1 = s3.Bucket(bucket_name1)
+print("created destination bucket\n")
+
+fileToBeUploaded = os.getcwd()+'/eml/email_sample.eml'
+key = 'email_sample'
+bucket.upload_file(fileToBeUploaded, key)
+
+for object in bucket1.objects.all():
+  print(object)
```

Comment:
@sanjeevsiva17 There was proper assertion was not added to ensure the attachment is presented in the destination bucket. Let us create a file name with some uuid and ensure that it is found in the destination bucket.